### PR TITLE
Many small improvements to Process docs

### PR DESCRIPTION
**Pull request by pavlos**

### CODE REVIEW

File: lib/elixir/lib/process.ex [hunk 1]
```diff
@@ -331,7 +330,7 @@ defmodule Process do
   end
 
   @doc """
-  The calling process starts monitoring the item given.
+  The calling process starts monitoring the `item` given.
```
Comment (whatyouhide) [inline lib/elixir/lib/process.ex]: We could take advantage of the PR and change this to "the given `item`", wdyt?

Comment (pavlos) [inline lib/elixir/lib/process.ex]: i like it

**Comment by pavlos** [comment]
@whatyouhide I make the changes you suggested.  Thanks for reviewing :)

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**